examples/demo-rollup/localsetup.md

```diff
@@ -0,0 +1,47 @@
+## Setting up SDK to run locally
+
+* Install docker https://www.docker.com
+* switch to the `demo-rollup` directory
+* Start the celestia services locally
+```
+make clean
+make start
```

Comment:
@dubbelosix, this looks super easy to use, but hard to understand. Can we add a thorough explanation of what's going on for each of these commands? 

Maybe something like this:

```markdown
## Setting up SDK to run locally

To start, you'll need to configure a Celestia client in your `rollup_config.toml`. The easiest way to get reliable access to a Celestia client is to spin up a local devnet - so we've provided a `makefile` to do that for you. If you want to configure your own Celestia client, simply paste its connection information into `rollup_config.toml`. Otherwise, follow the steps below to get started:


* Install docker https://www.docker.com
* switch to the `demo-rollup` directory
* Start the celestia services locally
...
```

Note - this can wait until after we've updated to `clap` if you want